build/build-image/Dockerfile

```diff
@@ -84,8 +86,9 @@ RUN mkdir /tmp/hugo && \
     mv /tmp/hugo/hugo /usr/local/bin/ && \
     rm -r /tmp/hugo
 
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+COPY /examples/nodejs-simple .
+RUN tar xvf node-v12.16.1-linux-x64.tar.gz
```

Comment:
Curious - any reason we're vendoring, and not pulling from a central website?

Reply:
The security folks recommend to directly checkin the package to avoid the curl|bash vulnerability.

Reply:
Where did node-v12.16.1-linux-x64.tar.gz come from - could we not download it from there?

I'm being cognizant of the fact that this repo (for better or worse), gets pulled in as a dependency for a go, rust, and terraform (probably others), and adding an extra 20MB to the repo for each version of node we want to vendor seems like something we should avoid?